### PR TITLE
Helm provider and Load Balancer controller

### PR DESCRIPTION
**Pull request by s-olabina**

### CODE REVIEW

File: infrastructure/modules/Helm/main.tf [hunk 1]
```diff
@@ -0,0 +1,34 @@
+module load-balancer-controller {
+  source  = "terraform-module/release/helm"
+  version = "~> 2.8.0"
+
+  namespace  = "vegait-load-balancer"
```
Comment (Igor-Knezevic-VegaIT) [inline infrastructure/modules/Helm/main.tf:5]: add parameter to create namespace if it does not exist.
At least that is needed for the resource, I don't know what module does by default if it doesnt exist.

File: infrastructure/modules/Helm/main.tf [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+  app = {
+    name          = "load-balancer"
+    version       = "1.8.1"
+    chart         = "aws-load-balancer-controller"
+    deploy        = 1
+  }
+
+
+  set = [
```
Comment (Igor-Knezevic-VegaIT) [inline infrastructure/modules/Helm/main.tf:18]: You are missing target type, check docs why you should change it.

Comment (Igor-Knezevic-VegaIT) [reply infrastructure/modules/Helm/main.tf:18]: You are missing region as well

File: infrastructure/modules/Helm/main.tf [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      name  = "serviceAccount.name"
+      value = "load-balancer"
+    },
+    {
+      name  = "serviceAccount.annotations.eks\\.amazonaws\\.com/role-arn"
+      value = var.iam_role_arn
+    },
+    {
+      name  = "serviceAccount.create"
+      value = "false"
```
Comment (Igor-Knezevic-VegaIT) [inline infrastructure/modules/Helm/main.tf]: This cannot be false as you need service account, make sure you understand how IRSA works.

File: infrastructure/modules/Helm/main.tf (hunk 1 above)
Comment (s-olabina) [reply infrastructure/modules/Helm/main.tf:5]: I will add it, module use false as default value.

File: infrastructure/main.tf [hunk 4]
```diff
@@ -27,7 +27,7 @@ provider "helm" {
     cluster_ca_certificate = base64decode(module.eks.cluster_ca_cert)
     exec {
       api_version = "client.authentication.k8s.io/v1beta1"
-      args        = ["eks", "get-token", "--cluster-name", module.eks.cluster_name]
+      args        = ["eks", "get-token", "--cluster-name", module.eks.cluster_name, "--output", "json", "--profile", var.profile]
```
Comment (Igor-Knezevic-VegaIT) [inline infrastructure/main.tf]: Why this?

**Review by Igor-Knezevic-VegaIT — APPROVED** [review]

**Pull request merged**